### ISSUE TITLE
no link in some feeds

### ISSUE THREAD

**Issue by heckj** [Fahrni/stream#27]
I have a feed that I regularly read with NNW that I tried in Stream, and it showed with no way to view the content.

The blog is <https://lapcatsoftware.com/articles/>, and the relevant article is <https://lapcatsoftware.com/articles/right-click.html>

The feed URL is http://lapcatsoftware.com/articles/atom.xml - There's new preview what-so-ever, but in NNW, the name of the blog itself is a link to the article, but not in Stream.

Summary of that ATOM feed:
```xml
<feed xmlns="http://www.w3.org/2005/Atom">
<title>The Desolation of Blog</title>
<author>
        <name>Jeff Johnson</name>
        <uri>https://lapcatsoftware.com/</uri>
</author>
<id>http://lapcatsoftware.com/articles/</id>
<link rel="self" type="application/atom+xml" href="https://lapcatsoftware.com/articles/atom.xml"/>
<link rel="alternate" type="text/html" href="https://lapcatsoftware.com/articles/index.html"/>
<updated>2020-09-27T18:15:00Z</updated>

<entry>
<title>Can't you just right click? Yes, with a workflow.</title>
<id>https://lapcatsoftware.com/articles/right-click.html</id>
<link rel="alternate" href="https://lapcatsoftware.com/articles/right-click.html"/>
<published>2020-09-27T18:15:00Z</published>
<updated>2020-09-27T18:15:00Z</updated>
</entry>

<entry>
<title>macOS Containers and defaults</title>
<id>https://lapcatsoftware.com/articles/containers.html</id>
<link rel="alternate" href="https://lapcatsoftware.com/articles/containers.html"/>
<published>2020-09-22T14:45:00Z</published>
<updated>2020-09-22T14:45:00Z</updated>
</entry>

</feed>
```

**Comment by heckj** [comment]
If it helps, here's a screenshot of what was displayed:
![IMG_B26EB98FD513-1](https://user-images.githubusercontent.com/43388/94496068-3cbbb480-01a8-11eb-8348-739f40dbe4e2.jpeg)

**Comment by Fahrni** [comment]
Fixed for 1.0.3
